client/js/src/main/scala/zio/zmx/client/frontend/views/DiagramView.scala

```diff
@@ -31,6 +41,58 @@ object DiagramView {
 
     private val titleVar = Var("")
 
+    def diagramControls(d: DiagramConfig): HtmlElement =
+      div(
+        cls := "flex w-full justify-around",
+        a(
+          cls := "text-4xl rounded text-center place-self-center",
+          "🔼 ",
```

Comment:
I would prefer to use icon buttons rather than UTF characters here. For now we can stick with what you have, but we should clean that up when we finalize the theming as in #330

Reply:
Yes, I agree this is only temporary until we get better UI from #330